cmake/projects/Lua/hunter.cmake

```diff
@@ -0,0 +1,23 @@
+if(DEFINED HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_ 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Lua
+    VERSION
+    "5.3.2"
+    URL
+    "https://github.com/hunter-packages/lua/archive/5.3.2.tar.gz"
```

Comment:
That's not so easy as it seems :) Many subtle details are revealed when you have to deal with multi-platform development. You make very good work. Thank you!